.github/env.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python = 3.10
```

Comment:
Why do we pin python 3.10? Could we not use >=3.7 or something?

Reply:
Initially I pinned this since I couldn't get planemo (actually setting up the galaxy instance) working with 3.11, your suggestion to update galaxy branch to 23.1 effectively fixed this: dependency is >=3.7 now